src/tools/compiletest/src/header.rs

```diff
@@ -555,8 +550,8 @@ pub fn lldb_version_to_int(version_string: &str) -> isize {
     let error_string = format!("Encountered LLDB version string with unexpected format: {}",
                                version_string);
     let error_string = error_string;
-    let major: isize = version_string.parse().ok().expect(&error_string);
-    return major;
+    let major: isize = version_string.parse().expect(&error_string);
+    major
```

Comment:
I believe this can be just one line, there's no need for the variable here. I guess it does indicate what we're getting from the version, but AIUI that's not actually important -- the isize is mostly opaque.

As a side note, which is out of scope for this PR, why is this an isize? Can the version number be negative? u32 or u64 would seem to make more sense...